From a4a4ed015d309cdedfcdda13bbdca1100cb78f37 Mon Sep 17 00:00:00 2001
From: Xinyu Chen <xinyu.chen@freescale.com>
Date: Thu, 1 Nov 2012 16:37:29 +0800
Subject: [PATCH 092/281] ENGR00232020 imx6: add watchdog dev node permission
 support

The /dev/watchdog device node is accessed by the
system_server, so system:root is set.

Change-Id: I1a789621c26ae5d6d3d672b61a57464c224cb36b
Signed-off-by: Xinyu Chen <xinyu.chen@freescale.com>
---
 imx6/etc/ueventd.freescale.rc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/imx6/etc/ueventd.freescale.rc b/imx6/etc/ueventd.freescale.rc
index 5795f20..f8345dc 100644
--- a/imx6/etc/ueventd.freescale.rc
+++ b/imx6/etc/ueventd.freescale.rc
@@ -21,6 +21,7 @@
 /dev/ion                  0660   media      system
 /dev/graphics/galcore	  0666 	 system     graphics
 /dev/graphics/fb0         0660   system     graphics
+/dev/watchdog             0660   system     root
 
 # sysfs properties
 /sys/devices/virtual/input/input*   name        0660  root   input
-- 
1.8.0

